### ISSUE TITLE
v167 Deprecates LinearTosRGB

### ISSUE THREAD

**Issue by ashconnell** [N8python/n8ao#29]
Running N8AO with Three 167 results in the following error:

```
THREE.WebGLProgram: Shader Error 0 - VALIDATE_STATUS false

Material Name: 
Material Type: ShaderMaterial

Program Info Log: Fragment shader is not compiled.


FRAGMENT

ERROR: 0:369: 'LinearTosRGB' : no matching overloaded function found
ERROR: 0:369: '=' : dimension mismatch
ERROR: 0:369: 'assign' : cannot convert from 'const mediump float' to 'out highp 4-component vector of float'


  364:         }
  365: #ifdef DITHERING
  366: 	gl_FragColor.rgb = dithering( gl_FragColor.rgb );
  367: #endif
  368:         if (gammaCorrection) {
> 369:             gl_FragColor = LinearTosRGB(gl_FragColor);
  370:         }
  371:     }
  372:   
```

This is because 167 removed the deprecated `LinearTosRGB` shader function.

It was removed here: https://github.com/mrdoob/three.js/commit/132f25879f58cbc27ef432499a54eb89c5da7e40

I'm not sure what the best fix for this is but hoping I saved you some time hunting down why :)

**Comment by N8python** [comment]
Will push a fix as soon as possible.

**Comment by N8python** [comment]
Fixed as of: n8ao@1.8.3
